crates/server/src/store/metrics.rs

```diff
@@ -31,7 +37,55 @@ impl PersistentStoreMetrics {
             "number of dns zone records in persisted storages"
         );
 
-        Self { zone_records }
+        #[cfg(feature = "__dnssec")]
+        let (
+            zone_records_dynamically_added,
+            zone_records_dynamically_deleted,
+            zone_records_dynamically_updated,
+        ) = {
+            let zone_records_dynamically_modified_name =
+                "hickory_zone_records_dynamically_modified_total";
```

Comment:
Nit: what is the `dynamically` part adding here? Just `added`/`deleted`/`updated`/`modified` seems to imply dynamicity.

Reply:
Removed the `dynamic` part as it does not add value.